File: Java/백준/Bronze/2525. 오븐 시계/오븐 시계.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = (sc.nextInt()) % 24;
        int b = sc.nextInt();
        b += sc.nextInt();

        if (b >= 60) {
            a = (a + b / 60) % 24;
            b %= 60;
        }

        System.out.printf("%d %d", a, b);
    }
}

File: Kotlin (JVM)/백준/Bronze/2525. 오븐 시계/오븐 시계.kt
import java.util.*

fun main() {
    val input = Scanner(System.`in`)
    val time = input.nextLine().split(" ").map { it.toInt() }
    val cookTime = input.nextInt()
    var h = time[0]
    var m = time[1]

    m = m + cookTime
    if (m >= 60) {
        h += m / 60
        m = m % 60
    }
    if (h >= 24) {
        h %= 24
    }
    println("$h $m")
}